go.mod

```diff
@@ -83,6 +83,7 @@ require (
 	github.com/firefart/nonamedreturns v1.0.4 // indirect
 	github.com/fogleman/gg v1.3.0 // indirect
 	github.com/fsnotify/fsnotify v1.5.4 // indirect
+	github.com/fullstorydev/grpcurl v1.8.6 // indirect
```

Comment:
Where is this used?

Reply:
It is indirect, go mod tidy added it. Not sure how to back-derive which dependency added the line.